electrum/ecc.py

```diff
@@ -374,6 +374,11 @@ def sign(self, data: bytes, sigencode=None, sigdecode=None) -> bytes:
             sigdecode = get_r_and_s_from_sig_string
         private_key = _MySigningKey.from_secret_exponent(self.secret_scalar, curve=SECP256k1)
         sig = private_key.sign_digest_deterministic(data, hashfunc=hashlib.sha256, sigencode=sigencode)
+        counter = 0
+        while ord(sig[0]) > 127: # grind for low R value https://github.com/bitcoin/bitcoin/pull/13666
```

Comment:
First, `sig` is bytes, so instead of `ord(sig[0])`, you want `sig[0]`.
But anyway, `sig` is in many cases DER encoded! It depends on `sigencode`. So this `while` will not be entered of course.
You should define something like
```
def sig_encode_r_s(r, s, order):
    return r, s
```
and pass that when to python-ecdsa, and only when r has the desired property, use the provided sigencode:
```
sig = sigencode(r, s)
```